stylus/ui-app/forms.styl

```diff
@@ -11,9 +11,11 @@ checkbox-size = 20px
 
 $form
     .coz-form
-        position relative
         margin   1em 0 .5em
 
+    .coz-form
+    .coz-form-group
```

Comment:
What is the use case for a new class? I can't really see the need without an example :(